src/plugins/intel_gpu/include/intel_gpu/plugin/device_config.hpp

```diff
@@ -47,6 +48,9 @@ struct Config {
         adjustKeyMapValues();
     }
 
+    uint32_t GetDefaultNStreamsForThroughputMode() const {
+        return 2;
+    }
```

Comment:
@myshevts @vladimir-paramuzov Replaced this as a function in Config. Please review this!

Reply:
I'm fine, but again, should this be a metric... maybe we can reuse the METRIC_KEY(RANGE_FOR_STREAMS)?
 the "higher" va;ue of that should match #CSSs... @vladimir-paramuzov whta do you think?